stripe/src/main/java/com/stripe/android/model/ShippingInformation.java

```diff
@@ -0,0 +1,78 @@
+package com.stripe.android.model;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.stripe.android.util.StripeNetworkUtils;
+
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.stripe.android.util.StripeJsonUtils.optString;
+import static com.stripe.android.util.StripeJsonUtils.putStringIfNotNull;
+
+/**
+ * Model representing a shipping address object
+ */
+public class ShippingInformation extends StripeJsonModel {
+
+    private static final String FIELD_ADDRESS = "address";
+    private static final String FIELD_NAME = "name";
+    private static final String FIELD_PHONE = "phone";
+
+    private @Nullable SourceAddress mAddress;
```

Comment:
I named it SourceAddress originally because it was a unique object from the Address fields in the Card object -- at the time I didn't know it was our "universal" address object. Is this the one that's going to get used everywhere but cards?